contracts/contracts/poolBooster/AbstractPoolBoosterFactory.sol

```diff
@@ -0,0 +1,174 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { Governable } from "../governance/Governable.sol";
+import { IPoolBooster } from "../interfaces/poolBooster/IPoolBooster.sol";
+import { IPoolBoostCentralRegistry } from "../interfaces/poolBooster/IPoolBoostCentralRegistry.sol";
+
+/**
+ * @title Abstract Pool booster factory
+ * @author Origin Protocol Inc
+ */
+contract AbstractPoolBoosterFactory is Governable {
+    struct PoolBoosterEntry {
+        address boosterAddress;
+        address ammPoolAddress;
+        IPoolBoostCentralRegistry.PoolBoosterType boosterType;
+    }
+
+    event PoolBoosterCreated(
+        address poolBoosterAddress,
+        address ammPoolAddress,
+        IPoolBoostCentralRegistry.PoolBoosterType poolBoosterType
+    );
+    event PoolBoosterRemoved(address poolBoosterAddress);
```

Comment:
Those two events seem to not be used anymore, no?

Reply:
done here: https://github.com/OriginProtocol/origin-dollar/pull/2400/commits/25f0d6277b87d1e493e4b34e7e232ee2c06cecca